packages/component-base/src/url-utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license
+ * Copyright (c) 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * Check if two paths match
+ *
+ * @param {string} path1
+ * @param {string} path2
+ */
+export function arePathsMatching(path1, path2) {
```

Comment:
Please, add TS declarations for this file.

Reply:
Done.